src/bindings.c

```diff
@@ -762,6 +762,11 @@ static int permute_prepare(void)
 		return -1;
 	}
 
+        if (!mkdir_p(new_runtime, 0755)) {
+		lxcfs_error("Failed to create dir %s\n", new_runtime);
+		return -1;
+        }
+
```

Comment:
What's going on with the indent here? Is it a mix of tabs/spaces?

Reply:
Yeah, maybe something funky with my editor. Switched to tabs like (most) of the rest of the code.